### PR TITLE
LPS-116987 Only show view link if url exists

### PR DESCRIPTION
**Pull request by carloslancha**
View icon should be only visible if viewURL exists

image

image

### CODE REVIEW

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
cc @dgarciasarai

**Comment by carloslancha** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js [hunk 1]
```diff
@@ -132,23 +133,28 @@ const SidebarPanelInfoView = ({
 								</p>
 
 								{sortedViewURLS.map((language) => (
-									<div
-										className="autofit-row autofit-row-center mb-1"
+									<ClayLayout.ContentRow
+										className="autofit-row-center mb-1"
```
Comment (dgarciasarai) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:137]: You can remove `autofit-row-center` and use api attribute `verticalAlign="center"` 😃

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js [hunk 2]
```diff
@@ -132,23 +133,28 @@ const SidebarPanelInfoView = ({
...
+										</ClayLayout.ContentCol>
 
-										<div className="autofit-col autofit-col-expand">
-											<div className="autofit-row autofit-row-center">
-												<div className="autofit-col inline-item-before small">
+										<ClayLayout.ContentCol
+											expand={!!language.viewURL}
+										>
+											<ClayLayout.ContentRow
+												className="autofit-row-center"
```
Comment (dgarciasarai) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:150]: Same here `verticalAlign="center"`

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js (hunk 1 above)
Comment (carloslancha) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:137]: I looked the docs and I didn't find it, that's my level of asleepness today

**Comment by carloslancha** [comment]
ci:stop

**Comment by carloslancha** [comment]
> ci:stop

I don't know why I keep trying to stop it, always ignores me

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js]: ContentRow section:

Screenshot 2020-07-24 at 12 45 22

**Comment by dgarciasarai** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:forward

**Pull request closed without merging**